src/main/java/org/fisco/bcos/web3j/codegen/SolidityFunctionWrapper.java

```diff
@@ -131,9 +131,15 @@ void generateJavaFiles(
             String destinationDir,
             String basePackageName,
             Map<String, String> addresses)
-            throws IOException, ClassNotFoundException {
+            throws IOException, ClassNotFoundException, UnsupportedOperationException {
         String className = Strings.capitaliseFirstLetter(contractName);
 
+        if (bin.length() > 0x40000) {
```

Comment:
Do not use a magic number in the code, define it as constant

Reply:
> Do not use a magic number in the code, define it as constant

done